spec/components/trainees/confirmation/personal_details/view_preview.rb

```diff
@@ -0,0 +1,27 @@
+require "faker"
```

Comment:
We cant use faker because in the review apps the rails env is production and so faker would not be bundle installed and so the previews would not work.